bin/main.rb

```diff
@@ -1 +1 @@
-puts "Hello world!"
+puts 'Hello world!'
```

Comment:
- [ ] Please ensure that main.rb is an executable file.
You can do this by adding `#!/usr/bin/env ruby` to the top of the file.

- [ ] Also, ensure executable file works (i.e. `bin/main.rb` should display the same message as `ruby bin/main.rb`) [screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/ruby/OOP/milestone1/execute_bin.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/59565018/)).

Reply:
Thank you for your review and also for your help last night. We have ensured that the main.rb file is an executable file. We have also made sure that the command for executable file works.